orttraining/orttraining/eager/ort_aten.cpp

```diff
@@ -10,6 +10,10 @@
 #include <c10/util/irange.h>
 #include <ATen/WrapDimUtils.h>
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-function"
+#include <ATen/native/ReduceOpsUtils.h>
+#pragma GCC diagnostic pop
```

Comment:
D:\a\_work\1\s\orttraining\orttraining\eager\ort_aten.cpp(13,9): error C2220: the following warning is treated as an error [D:\a\_work\1\b\Release\onnxruntime_pybind11_state.vcxproj]
D:\a\_work\1\s\orttraining\orttraining\eager\ort_aten.cpp(13,9): warning C4068: unknown pragma 'GCC' [D:\a\_work\1\b\Release\onnxruntime_pybind11_state.vcxproj]
D:\a\_work\1\s\orttraining\orttraining\eager\ort_aten.cpp(14,9): warning C4068: unknown pragma 'GCC' [D:\a\_work\1\b\Release\onnxruntime_pybind11_state.vcxproj]
D:\a\_work\1\s\orttraining\orttraining\eager\ort_aten.cpp(16,9): warning C4068: unknown pragma 'GCC' [D:\a\_work\1\b\Release\onnxruntime_pybind11_state.vcxproj]